src/utils.jl

```diff
@@ -25,3 +25,12 @@ function line_to_tree(line::Vector)
   end
   return [line_to_tree(line[1:(end - 1)]), line[end]]
 end
+
+function Base.union(
+  graph1::AbstractNamedGraph,
+  graph2::AbstractNamedGraph,
+  graph3::AbstractNamedGraph,
+  graph_rest::AbstractNamedGraph...,
+)
+  return union(union(graph1, graph2), graph3, graph_rest...)
+end
```

Comment:
Could you move this to a PR in `NamedGraphs.jl`?

Reply:
Thanks, should be available in NamedGraphs v0.1.20 now (we should make that the lower bound of the `[compat]` entry in `Project.toml`).